### ISSUE TITLE
Emergency box for non-humans

### ISSUE THREAD

**Issue by Drakkon** [yogstation13/yogstation#2064]
In addition to oxy tank, mask, flare, and medipen, plant people get a flashlight and androids a basic cell.

**Comment by ShadowDeath6** [comment]
races have specific benefits and drawbacks
if you want your roundstart shit go look for it in tool storage at roundstart :-1:

**Comment by doerakpoes** [comment]
Jelly seems pissed off about this idea

**Comment by drderp3635** [comment]
Plant people already start with a flashlight in thier pda, and humans don't start with emergency food so why should androids

**Comment by Drakkon** [comment]
\/\/3ll fu|< u 2 m8 :'c

**Comment by Lightning38088** [comment]
Itd just be better if the meme race known as plasmashits get a extra tank.

**Comment by ShadowDeath6** [comment]
nanotrasen designs AIs that control the station to ignore the commands and well being of non humans don't say they're "not dumb enough" to waste money on giving their robot people batteries roundstart

**Comment by MacHac** [comment]
Everyone needs to acknowledge the real problem here, which is that plasmamen don't have a suit storage spot to put their plasma tank.  I say, give them a special jumpsuit with the extinguishers, and move their standard outerwear to the suit slot with room for a plasma tank.  That way, taking plasmaman doesn't leave you with either 1 hand or no backpack.

**Comment by Drakkon** [comment]
@ShadowDeath6 Was hoping the AI problem would be saved for another discussion, but ya know... they should at least be acknowledged with minor subhuman rights.
"waste money"... "basic cell"... -_-

**Comment by ShadowDeath6** [comment]
It kind of is wasting money when they havs to give them extra stuff that's already found in abundance on the station

**Comment by Mrcreepssss** [comment]
yeah, and RnD can always make more batteries, though i like the idea of this species specific starting shit,

**Comment by The-clown-known-as-Pepe** [comment]
I mean you already get oxygen tanks that you can kill yourself with if you play non-op race (which is human, git gud)

**Comment by Mrcreepssss** [comment]
humans are probably the most powerfull, not by sheer strength but in head roles and AI's loyalty.

**Comment by The-clown-known-as-Pepe** [comment]
Humans are most powerfull because they have no drawbacks.

**Comment by Mrcreepssss** [comment]
then why did you say otherwise?

**Comment by Mrcreepssss** [comment]
@MacHac sounds pretty good but what about EVA? like this they wouldn't be able to EVA at all, unless you were to make their suit spaceproof by standard, but that would be pretty OP though they can't really be rezzed either? also wouldn't this make them unstrippable without killing/wounding them?
